c2corg_api/views/user.py

```diff
@@ -53,6 +54,8 @@ def validate_json_password(request):
         # later on requires plain string otherwise it raises
         # the "Unicode-objects must be encoded before hashing" error.
         password = request.json['password']
+        if len(password) < 3:
+            raise 'Password too short'
```

Comment:
When you `raise ...`, does this also add the error message as with `request.errors.add('body', 'password', 'Required')`?